pkg/cloudagents/upload.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 LiveKit, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudagents
+
+import (
+	"bytes"
+	"embed"
+	"fmt"
+	"io"
+	"io/fs"
+	"mime/multipart"
+	"net/http"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+var (
+
+	//go:embed defaults/*
+	embedfs embed.FS
```

Comment:
We might want to expose this as part of the public API, I think we want to use some of these files for initial codegen if creating an agent with CLI that does not come from a template that already has these files included?

Reply:
Could you elaborate what we want to expose? The actual default ignore files?

Reply:
Yes, default .dockerignore files. I would also move the default Dockerfiles here too so they're co-located and easy to update together.

Reply:
But I can also see the argument for not embedding these files in everyone's copy of the SDK even when not using cloud agents